src/task.js

```diff
@@ -0,0 +1,10 @@
+export default class Task {
+  constructor(desc, index) {
+    this.desc = desc;
+    this.completed = false;
+    this.index = index;
+    this.id = Task.generateID();
```

Comment:
This part needs to be refactored because the index and id are essentially the same.